src/events/endpoints.go

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright © 2020, Oracle and/or its affiliates. All rights reserved.
+ * The Universal Permissive License (UPL), Version 1.0
+ */
+package events
+
+import (
+	"github.com/go-kit/kit/endpoint"
+	"github.com/go-kit/kit/tracing/opentracing"
+	stdopentracing "github.com/opentracing/opentracing-go"
+	"golang.org/x/net/context"
+)
+
+// Endpoints collects the endpoints that comprise the Service.
+type Endpoints struct {
+	EventsEndpoint endpoint.Endpoint
```

Comment:
nit: these should probably just be named `Events` and `Health` as they already live inside the Endpoints struct (i.e. no need to repeat the Endpoint again)